libs/cardano-ledger-core/src/Cardano/Ledger/Val.hs

```diff
@@ -66,6 +67,12 @@ class
   -- | If a quantity is stored in only one of 'v1' or 'v2', we use 0 for the missing quantity.
   pointwise :: (Integer -> Integer -> Bool) -> t -> t -> Bool
 
+  isAdaOnly :: t -> Bool
```

Comment:
the only trouble with renaming this from `adaOnly` to `isAdaOnly` is that both the ShelleyMA and the Alonzo spec are using `adaOnly`.

Reply:
Would it be problematic if we renamed it in the spec as well? `isAdaOnly` is a lot more descriptive name. It would be much better if we could improve the spec as well as the code. So, if it is possible I can go through and change it in the spec, but if it is a problem for some reason, I can definitely just move the `adaOnly` function into the `Val` class.

Reply:
nope, renaming it in the spec is my favorite option (I agree that `isAdaOnly` is more descriptive).

Reply:
Renamed.